src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__img img-js"
+        src="./images/imac.jpeg"
+        alt="imac computer"
+      >
+      <div class="product-card__about">
+        <p class="product-card__about-description about-description-js">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        <p class="product-card__about-code">Код товару: <span class="about-code-js">195434</span></p>
+      </div>
+      <div class="product-card__rating">
+        <div class="product-card__rating-stars">
```

Comment:
```suggestion
        <div class="product-card__rating stars stars--4">
```
stars - отдельный блок
stars--4 модификатор, отвечающий за количество активных звезд